### PR TITLE
updated readme, yml files and Test_Bitbucket playbook

### PR DESCRIPTION
**Pull request by RotemAmit**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.yml [hunk 1]
```diff
@@ -1324,7 +1324,7 @@ script:
         - contextPath: Bitbucket.PullRequest.source.repository.full_name
           description: The full name of the repository of the source branch.
           type: String
-        - contextPath: Bitbucket.PullRequest.source.repository.links.self.href
+        - contextPath: Bitbucket.PullRequest.source.repository.links
```
Comment (Shellyber) [inline Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.yml:1327]: You'll need to revert this. This is breaking backwards.
Add the new path that you want instead

File: Packs/Bitbucket/ReleaseNotes/1_0_1.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Bitbucket
+- Updated the Test_Bitbucket playbook so that the first task will be DeleteContext.
```
Comment (Shellyber) [inline Packs/Bitbucket/ReleaseNotes/1_0_1.md:4]: Remove this. Changes in the test playbook shouldn't be reflected

File: Packs/Bitbucket/ReleaseNotes/1_0_1.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+- Updated the yaml file so that the output of the pull request commands will have information about the reviewers of the PR.
```
Comment (Shellyber) [inline Packs/Bitbucket/ReleaseNotes/1_0_1.md:5]:
```suggestion
- Updated the outputs of the pull request commands with the information on the reviewers.
```

File: Packs/Bitbucket/ReleaseNotes/1_0_1.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+- Updated the README file according to the changes in the yaml file, and added a command example to the command bitbucket-pull-request-update.
```
Comment (Shellyber) [inline Packs/Bitbucket/ReleaseNotes/1_0_1.md:6]:
```suggestion
- General documentation improvements.
```

**Review by Shellyber — COMMENTED** [review]
Nice

File: Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.yml (hunk 1 above)
Comment (RotemAmit) [reply Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.yml:1327]: And have two contextPaths? 
It is supposed to be without .self.href, I probably missed it before

Comment (Shellyber) [reply Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.yml:1327]: Yes, we will have 2. one with self.href and one only with links

**Review by Shellyber — APPROVED** [review]

**Pull request merged**